qa/index.html

```diff
@@ -20,76 +20,78 @@
       accept=".doc,.docx,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document"
     />
 
-    <button type="button" onclick="trackChangesWindow()">
-      Track Changes Window
-    </button>
     <button type="button" onclick="toggleTrackChanges()">
       TOGGLE TRACK CHANGES
     </button>
     <button type="button" onclick="acceptTrackChange()">Accept TC</button>
     <button type="button" onclick="rejectTrackChange()">Reject TC</button>
+    <button type="button" onclick="gotoOutline()">Goto Outline</button>
     <script src="wrapper.js"></script>
     <office-doc id="el-embed" />
     <script>
-      const picker = document.getElementById('el-picker');
```

Comment:
We should add a prettierrc file to project it seems, don't want these insignificant changes polluting the diff.

Reply:
Agreed. Let's create a separate formatting PR for that